_inc/client/security/loading-card.jsx

```diff
@@ -0,0 +1,23 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { translate as __ } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import SettingsCard from 'components/settings-card';
+import SettingsGroup from 'components/settings-group';
+
+export function LoadingCard( props ) {
```

Comment:
It would be useful to list the expected `props` and their respective `propTypes` here.